src/domain/collectibles/entities/schemas/collectible.schema.ts

```diff
@@ -1,40 +1,17 @@
-import { JSONSchemaType } from 'ajv';
-import { Collectible } from '@/domain/collectibles/entities/collectible.entity';
+import { z } from 'zod';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
 
-export const COLLECTIBLE_SCHEMA_ID =
-  'https://safe-client.safe.global/schemas/collectibles/collectible.json';
-
-export const collectibleSchema: JSONSchemaType<Collectible> = {
-  $id: COLLECTIBLE_SCHEMA_ID,
-  type: 'object',
-  properties: {
-    address: { type: 'string' },
-    tokenName: { type: 'string' },
-    tokenSymbol: { type: 'string' },
-    logoUri: { type: 'string' },
-    id: { type: 'string' },
-    // Do not use format: 'uri' as it fails on some payloads that should be considered valid
-    uri: {
-      oneOf: [{ type: 'string' }, { type: 'null', nullable: true }],
-      default: null,
-    },
-    name: {
-      oneOf: [{ type: 'string' }, { type: 'null', nullable: true }],
-      default: null,
-    },
-    description: {
-      oneOf: [{ type: 'string' }, { type: 'null', nullable: true }],
-      default: null,
-    },
-    // Do not use format: 'uri' as it fails on some payloads that should be considered valid
-    imageUri: {
-      oneOf: [{ type: 'string' }, { type: 'null', nullable: true }],
-      default: null,
-    },
-    metadata: {
-      oneOf: [{ type: 'object' }, { type: 'null', nullable: true }],
-      default: null,
-    },
-  },
-  required: ['address', 'tokenName', 'tokenSymbol', 'logoUri', 'id'],
-};
+export const CollectibleSchema = z.object({
+  address: AddressSchema,
+  tokenName: z.string(),
+  tokenSymbol: z.string(),
+  logoUri: z.string(),
+  id: z.string(),
+  // AJV remnant - do not use format: 'uri' as it fails on some payloads that should be considered valid
```

Comment:
Nit: do you think the AJV reference is needed? We could just comment out that we don't enforce URL validation here, wdyt?

Reply:
I changed the comment in 01237f555881832ebe717e380095241ffb44c2a1 but I think we should revisit this in the future. If we can make the schemas stricter, it would be beneficial.